docs/conf.py

```diff
@@ -48,7 +48,7 @@ def noop_decorator(*args, **kwargs):
 export_config_help(configs_file_path)
 
 project = "Modin"
-copyright = "2018-2021, Modin"
+copyright = "2018-2022, Modin"
```

Comment:
is there any reason this is "Modin" and not "Modin Developers" or vice versa?

Reply:
This copyright is shown in our documentation. "Modin Developers" would look strange to me.
![image](https://user-images.githubusercontent.com/26389732/149078010-04dbd4be-b5b0-4747-8589-e98df6826d55.png)

Reply:
I think it is weird that we have `Modin Developers` in the `NOTICE` file, but not here, we can change it in a new PR.

@jeffreykennethli do you want to open a new issue for it?

Reply:
opened https://github.com/modin-project/modin/issues/3962